Explorer/Assets/DCL/MapRenderer/MapLayers/UsersMarker/HotUserMarker.cs

```diff
@@ -25,6 +27,8 @@ internal HotUserMarker(IObjectPool<HotUserMarkerObject> pool, ICoordsUtils coord
             poolableBehavior = new MapMarkerPoolableBehavior<HotUserMarkerObject>(pool);
         }
 
+        public void ToggleSelection(bool isSelected) { }
```

Comment:
Since you are using empty interface implementations in this whole shape, maybe consider using it here as well not to have empty implementations.